fix_sizes.py

```diff
@@ -0,0 +1,79 @@
+import maya.cmds as mc
+
+#file_list = "C:/projects/Perforce/Alex_Workspace1/Asset-Master/Michael Foods/Michael Foods Project Joey/Test Cell 1/Assets/HighRes/measurements.txt"
+#file_dir = "C:/projects/Perforce/Alex_Workspace1/Asset-Master/Michael Foods/Michael Foods Project Joey/Test Cell 1/Assets/HighRes/"
+file_list = "U:/Craig.LaMorte/Fix_size" #file path for the document containing the products and resizes
+file_dir = "U:/Craig.LaMorte/Fix_size/models/" #file path for the maya files
+
+product_keys = []
+desired_heights = []
+desired_widths = []
+desired_depths = []
+
+
+try:
+    dimensions = open(file_list, "rb") #opens file with all the resizes
+except IOError:
```

Comment:
If you are trying to throw an exception but add a custom message to it, you can use

except IOError as e:
    msg = 'Some message'
    raise IOError(msg)